terraform/revisions.tf

```diff
@@ -0,0 +1,22 @@
+locals {
+  mysql_revisions = {
+    amd64 = 180, # renovate tag
```

Comment:
Renovate rules would apply here

Reply:
CC: @lucasgameiroborges

Reply:
would all the revision pins need a `# renovate tag` comment?

Reply:
that's a placeholder because I don´t know how to do it, forgot to remove